Added functionality to the filter

narrat32 opened the pull request.

narrat32 committed:
Added filter for trees

Added a filter for the trees in the shop

narrat32 committed:
Added a filter for the related products section

KadinHoneyfield reviewed (changes requested):
Nice work so far! Downloaded branch and tested. But, I can't seem to deselect a filter option? So when I was testing it I found it hard to actually find a combination that matches a tree as all filters need to be filled out. Are we able to add an "All" option to the drop down? Otherwise marker might never make a correct filter combination.

narrat32 committed:
Refined filter functionality

narrat32 committed:
Fixed filter issues

Known issues:
- will display extra results based on inputted values
- Price and height functionality doesn't work

KadinHoneyfield reviewed (approved):
Sweet, that looks like it has fixed most of it! We'll need to look at the price and height in future.
